rep-2006.rst

```diff
@@ -0,0 +1,71 @@
+REP: 2006
+Title: ROS 2 Vulnerability Disclosure Policy
+Author: ROS 2 Security Working Group, Chris Lalancette <clalancette@openrobotics.org>
+Status: Active
+Type: Informational
+Content-Type: text/x-rst
+Created: 12-May-2020
+Post-History:
+
+
+Abstract
+========
+
+This REP describes the Vulnerability Disclosure Policy related to the ROS 2 Common Packages as defined in REP 2005.
+
+
+Motivation
+==========
+
+The developers of the Robot Operating System (ROS) take security very seriously.
```

Comment:
I did think about that.  The thing is that I think we are "developers of ROS", its just that the scope of what is covered in this document is ROS 2.  So I'd prefer to leave this as-is.

Reply:
It's just such a common thing to hear in the community "ROS development didn't care about security" that it seems a low hanging fruit to disambiguate here for a potentially significant perception impact. We do know this REP only applies to ROS 2 but it can be overlooked and consider REPs as applying to the entire ROS ecosystem. For example REP 2004 has ROS 2 in mind but doesn't explicitly call out its scope to be ROS 2 and can thus be applied to ROS 1.

Reply:
The difference here is that we explicitly call out the scope to be REP-2005, so in my thinking there isn't really any ambiguity about what is covered.

Reply:
I know we don't agree on this, but I don't see a way out of the current stalemate since both sides are a matter of opinion.  I'm going to resolve this conversation to move forward, but feel free to reopen or keep commenting.